build.gradle

```diff
@@ -128,8 +127,9 @@ dependencies {
 
   /* config */
   implementation 'com.typesafe:config:1.4.3'
-  implementation "com.github.carueda:tscfg_2.13:$tscfgVersion"
   implementation "com.github.scopt:scopt_${scalaVersion}:4.1.0" // cmd args parser
+  implementation "com.github.pureconfig:pureconfig_${scalaVersion}:${pureConfigVersion}"
+  implementation "com.github.pureconfig:pureconfig-pekko_${scalaVersion}:${pureConfigVersion}"
```

Comment:
`pureconfig-pekko` seems to be not used. You could also remove the `pureConfigVersion` variable then